Makefile

```diff
@@ -28,7 +28,7 @@ ifndef DOCKER_HUB_REGISTRY_IMG
     $(warning DOCKER_HUB_REGISTRY_IMG not defined, using '$(DOCKER_HUB_REGISTRY_IMG)' instead)
 endif
 ifndef BASE_REGISTRY_IMG
-    BASE_REGISTRY_IMG := docker.io/portworx/px-operator-registry:23.3.1
+    BASE_REGISTRY_IMG := docker.io/portworx/px-operator-registry:23.5.0
```

Comment:
This is the base registry on top of which we add our current new (23.5.0 in this case) operator and then build a new registry with tag 23.5.0.
This should point to the latest released operator version `23.3.1` right now. Once `23.4.0` is release, we will change this again to point to `23.4.0`.